client/cmd/resource/apply.go

```diff
@@ -0,0 +1,157 @@
+package resource
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/goto/salt/log"
+	"github.com/spf13/cobra"
+
+	"github.com/goto/optimus/client/cmd/internal/connectivity"
+	"github.com/goto/optimus/client/cmd/internal/logger"
+	"github.com/goto/optimus/client/cmd/internal/progressbar"
+	"github.com/goto/optimus/client/cmd/internal/survey"
+	"github.com/goto/optimus/config"
+	pb "github.com/goto/optimus/protos/gotocompany/optimus/core/v1beta1"
+)
+
+const (
+	applyTimeout  = time.Minute * 5
+	successStatus = "success"
+)
+
+type applyCommand struct {
+	logger log.Logger
+
+	configFilePath string
+	clientConfig   *config.ClientConfig
+
+	namespaceSurvey *survey.NamespaceSurvey
+	namespaceName   string
+	projectName     string
+	storeName       string
+
+	verbose       bool
+	resourceNames []string
+}
+
+// NewApplyCommand initializes command for applying resources from optimus to datastore
+func NewApplyCommand() *cobra.Command {
+	l := logger.NewClientLogger()
+	apply := &applyCommand{
+		logger:          l,
+		namespaceSurvey: survey.NewNamespaceSurvey(l),
+	}
+
+	cmd := &cobra.Command{
+		Use:     "apply",
```

Comment:
Instead of `apply`, why don't we use `sync`? Asking because in most of the operation underneath (excluding the handler), it's worded as `sync` instead `apply`. And in the description between the short and long one is a bit different.

Reply:
Sure, will update the short description.